### DISCUSSION TITLE
how can i run an existing Symfony project locally

### DISCUSSION THREAD

**Discussion by TADBIRAL** [symfony/symfony#45898] [Q&A]
Hello,
i've download the whole project but i don't know how can i run it in local.
Can anyone please tell how ?

PS : this project was made with an outdated symfony version (symfony2) so if anyone have any idea how can i upgrade it that would be a plus,
THANK YOU

**Comment by alessandro-podo** [comment]
for upgrade: https://symfony.com/doc/2.7/setup/upgrade_major.html
you hava a webserver with php installed local?

**Comment by GromNaN** [comment]
So many things changed from 2.x to 6.x. The simplest strategy is to create a new app and import code, feature by feature, from the old app.

To run locally, have you tried the local server provided by symfony cli?
https://symfony.com/doc/current/setup/symfony_server.html
